app/constants/browse_channels.ts

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+export const PUBLIC = 'public';
```

Comment:
are we expecting to use this constants somewhere else other than the `BrowseChannels` screen? does it make sense to call either call the file `channels` or have the constants directly in the drop down component and export it?